examples/get-started/contour-map.py

```diff
@@ -0,0 +1,130 @@
+# the colorbar, in this case every 1,000 meters. To set the label for an axis
+# on the colorbar, the value begins with either "x+l" (x-axis) or "y+l"
+# (y-axis), followed by the intended label.
+#
+# By default, the CPT for the colorbar is the same as the one set
+# in :meth:`pygmt.Figure.grdimage`.
+
+fig = pygmt.Figure()
+fig.grdimage(grid=grid, frame="a", projection="M10c", cmap="haxby")
+fig.colorbar(frame=["a1000", "x+lElevation", "y+lm"])
+fig.show()
+
+###############################################################################
+# Adding contour lines
+# --------------------
+#
+# To add contour lines to the color coded figured, the
+# :meth:`pygmt.Figure.grdcontour` method is used. The ``frame`` and
+# ``projection`` are already set using :meth:`pygmt.Figure.grdimage` and are
+# not needed again. However, the same input for ``grid`` (in this case, the
+# variable named "grid") must be input again. The ``interval`` parameter sets
+# the spacing between lines (in this case, 500 meters), and the ``annotation``
+# parameter draws darker lines that are annotated with the elevation or
+# bathymetry.
+
+fig = pygmt.Figure()
+fig.grdimage(grid=grid, frame="a", projection="M10c", cmap="haxby")
+fig.grdcontour(grid=grid, interval=500, annotation=1000)
+fig.colorbar(frame=["a1000", "x+lElevation (m)"])
+fig.show()
+
+###############################################################################
+# Color in land
+# -------------
+#
+# To make it clear where the islands are located, the
+# :meth:`pygmt.Figure.coast` method can be used to color in the land. The
+# ``land`` is colored in as "lightgray", and the ``shorelines`` parameters
+# draws a border around the islands.
+
+fig = pygmt.Figure()
+fig.grdimage(grid=grid, frame="a", projection="M10c", cmap="haxby")
+fig.grdcontour(grid=grid, interval=500, annotation=1000)
+fig.coast(shorelines="2p", land="lightgray")
+fig.colorbar(frame=["a1000", "x+lElevation (m)"])
+fig.show()
+
+###############################################################################
+# Additional exercises
+# --------------------
+#
+# This is the end of the second tutorial. Here are some additional exercises
+# for the concepts that were discussed:
+#
+# 1. Change the resolution of the grid file to either "01m" (lower resolution)
+#    or "15s" (higher resolution). Note that higher resolution grids will have
+#    larger file sizes.
+#
+# 2. Create a contour map of the area around Mt. Rainier. A suggestion for the
+#    ``region`` would be ``[-122, -121, 46.5, 47.5]``. Adjust the
+#    :meth:`pygmt.Figure.grdcontour` and :meth:`pygmt.Figure.colorbar`
+#    settings as needed to make the figure look good.
+#
+# 3. Create a contour map of São Miguel Island in the Azores; a suggested
+#    ``region`` is ``[-26, -25, 37.5, 38]``. Instead of coloring in ``land``,
+#    set ``water`` to "lightblue" to only display Earth relief information for
+#    the land.
+#
+# 4. Try other CPTs, such as "jet" and "geo".
```

Comment:
No jet please...
```suggestion
# 4. Try other CPTs, such as "SCM/fes" or "geo".
```

Reply:
Happy to change it, but what is the problem with jet?

Reply:
> Happy to change it, but what is the problem with jet?

@weiji14  I hope it is OK that I comment on this question.

@willschlitzer  This is an article addressing the issues with the `jet` colormap, including some visual examples:
Crameri, F., G.E. Shephard, and P.J. Heron (2020), The misuse of colour in science communication, Nature Communications, 11, 5444. https://www.nature.com/articles/s41467-020-19160-7.

Reply:
Good to know! Thanks!